### PR TITLE
#310 - Add TCK test to validate that @Fallback can be used alone

### PR DESCRIPTION
**Pull request by brunobat**
Apart from the TCK tests I've added a clarification to the documentation.

Signed-off-by: brunobat <bbaptista@tomitribe.com>

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

File: spec/src/main/asciidoc/fallback.asciidoc [hunk 1]
```diff
@@ -21,7 +21,9 @@
 
 [[fallback]]
 == Fallback
-Fallbacks are invoked once a Retry or CircuitBreaker has failed enough times.
+Fallback provides an graceful alternative execution in case of failure.
+
+Fallbacks are invoked when used alone, once a Retry or CircuitBreaker has failed enough times.
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/fallback.asciidoc]: This line should be reworded
Fallback method is invoked when the method with `@Fallback` annotation completed exceptionally. `Fallback` annotation can be used on its own or together with other Fault Tolerance annotations.

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java [hunk 2]
```diff
@@ -253,4 +256,38 @@ public void testFallbackMethodWithArgsSuccess() {
...
         
     }
+
+    /**
+     * Test that a Fallback service can work alone, without other annotations.<br>
+     * <p>
+     * ServiceA fallback is driven by a class implementing FallbackHandler. ServiceA is only called once.
+     */
+    @Test
+    public void testStandaloneClassLevelFallback() {
```
Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java]: The annotation `Fallback` is method level annotation only as it is less useful to be on class level.

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/fallback/clientserver/FallbackOnlyClient.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
```
Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/fallback/clientserver/FallbackOnlyClient.java]: should be 2018

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java (hunk 2 above)
Comment (Emily-Jiang) [reply tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java]: ah. You meant `Fallback` handler. please change the method name to be testStandaloneHandlerFallback

**Comment by carlosdlr** [comment]
looks good @brunobat

**Pull request merged**